CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+- Migrate the build Installation Assistant files jenkins workflow to GHA ([#77](https://github.com/wazuh/wazuh-installation-assistant/pull/77))
```

Comment:
Consider changing the name of the PR and, then, this entry of the `CHANGELOG.md` as we are not migrating anything here, this automation is new.

Reply:
Fixed in https://github.com/wazuh/wazuh-installation-assistant/pull/77/commits/aa18f6c81eb4aae8d54dab76e8dfdb18d651545e